src/controllers/controllerengine.cpp

```diff
@@ -1440,6 +1440,23 @@ void ControllerEngine::softTakeover(QString group, QString name, bool set) {
 }
 
 /*  -------- ------------------------------------------------------
+     Purpose: Ignores the next value for the given ControlObject
+                This is used when an absolute physical control is changed to
```

Comment:
ok good say that too. Describing when the function should be called is helpful, it's hard to understand otherwise.